docs/reference/mapping/types/numeric.asciidoc

```diff
@@ -137,7 +139,26 @@ The following parameters are accepted by numeric types:
 
     Accepts a numeric value of the same `type` as the field which is
     substituted for any explicit `null` values.  Defaults to `null`, which
-    means the field is treated as missing.
+    means the field is treated as missing.  Note that this cannot be set
+    if the `script` parameter is used.
+
+`on_script_error`::
+
+    Defines what to do if the script defined by the `script` parameter
+    throws an error at indexing time.  Accepts `reject` (default), which
+    will cause the entire document to be rejected, and `ignore`, which
+    will register the field in the document's
+    <<mapping-ignored-field,`_ignored`>> metadata field and continue
+    indexing.  This parameter can only be set if the `script` field is
+    also set.
+
+`script`::
+
+    If this parameter is set, then the field will index values generated
+    by this script, rather than reading the values directly from the
```

Comment:
The `rather` suggests that values that are set on the document would always be ignored, is it correct? I wonder if we should make it more explicit.

Do we have tests for this case (I couldn't find them from a quick look but I might have just missed them), as well as for the case when the script computes values based on content of the document under the _same_ field name?

Reply:
We reject documents that have a value for the field, if the field declares a script. We should probably clarify that in the docs.

Reply:
+1 to explicitly documenting this, I will open a followup